src/ngraph/runtime/executable.hpp

```
@@ -69,6 +69,10 @@ class ngraph::runtime::Executable
     /// \returns an ngraph::ResultVector of all input parameters
     const ngraph::ResultVector& get_results() const;
 
+    /// \brief Get the default pipeline_depth for this executable
+    /// \returns default pipeline_depth
+    virtual size_t get_default_pipeline_depth() const;
```

What does default depth mean? Is it the preferred depth?

---

@rkimballn1  "default" is misleading, changed to "preferred"; also addressed other feedback. thanks!